content/docs/command-reference/remove.md

```diff
@@ -30,12 +30,7 @@ how it can be used to replace or modify files that are tracked by DVC.
 
 ## Options
 
-- `-o`, `--outs` - remove the outputs described in the given `targets`, keep the
-  `.dvc` files themselves. **This is the default behavior.**
-
-- `-p`, `--purge` - remove outputs and `.dvc` files.
-
-- `-f`, `--force` - force purge. Skip confirmation prompt.
+- `--outs` - remove outputs as well.
```

Comment:
Hm let's keep the previous description?
```suggestion
- `--outs` - remove the outputs described in the given `targets`.
```

Reply:
Once again, I used message identical with core repo. Probably `--outs` description should be applied to core repo first,

Reply:
No don't worry about that, same explanation as in https://github.com/iterative/dvc.org/pull/1497#discussion_r446080399.